asm/non_matchings/code_80005FD0/update_player_path_completion.s

```diff
@@ -67,20 +67,20 @@ glabel func_8000929C
 /* 009F74 80009374 24010014 */  li    $at, 20
 /* 009F78 80009378 30E5FFFF */  andi  $a1, $a3, 0xffff
 /* 009F7C 8000937C 14410005 */  bne   $v0, $at, .L80009394
-/* 009F80 80009380 3C068016 */   lui   $a2, %hi(D_80163448) # $a2, 0x8016
-/* 009F84 80009384 0C002E57 */  jal   func_8000B95C
-/* 009F88 80009388 8CC63448 */   lw    $a2, %lo(D_80163448)($a2)
+/* 009F80 80009380 3C068016 */   lui   $a2, %hi(gActualPath) # $a2, 0x8016
+/* 009F84 80009384 0C002E57 */  jal   update_player_position_factor
+/* 009F88 80009388 8CC63448 */   lw    $a2, %lo(gActualPath)($a2)
```

Comment:
if actual path refers to the path the AI use. We should make this clear this is like the gMainPath or gCoursePath or gAIPath or something like that.

Reply:
not sure I think it's a little more general then just AI

Reply:
This is being set to a path index of a player.

gPlayerPathIndex or something